docs/atlascli/command/atlas-clusters-availableRegions.txt

```diff
@@ -0,0 +1,59 @@
+.. _atlas-clusters-availableRegions:
+
+===============================
+atlas clusters availableRegions
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Manage available regions for your project.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for availableRegions
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Profile to use from your configuration file.
+
+Related Commands
```

Comment:
Can we include an example here? Not sure what managing available regions would look like so may be helpful

Reply:
Wait i think I added few examples in docs/atlascli/command/atlas-clusters-availableRegions-list.txt, is this what you were looking for or something else?

Reply:
Oh my bad, nevermind :)